airbyte-integrations/connectors/source-hubspot/unit_tests/test_field_type_converting.py

```diff
@@ -75,6 +75,7 @@ def test_bad_field_type_converting(field_type, expected, capsys):
         (["null", "number"], "user_id", "123", 123),
         (["null", "string"], "some_field", "123", "123"),
         (["null", "string"], "some_field", "", ""),
+        (["null", "number"], "null_field", "", 0),  # 1
```

Comment:
remove the comment, or add meaningful one

Reply:
Done.